src/probnum/problems/zoo/linalg/_random_spd_matrix.py

```diff
@@ -97,16 +97,14 @@ def random_sparse_spd_matrix(
     density: float,
     chol_entry_min: float = 0.1,
     chol_entry_max: float = 1.0,
-    format="csr",
+    format="csr",  # pylint: disable="redefined-builtin"
     random_state: Optional[RandomStateArgType] = None,
 ) -> np.ndarray:
-    """Random sparse symmetric positive definite matrix.
+    r"""Random sparse symmetric positive definite matrix.
 
     Constructs a random sparse symmetric positive definite matrix for a given degree
     of sparsity. The matrix is constructed from its Cholesky factor :math:`L`. Its
-    diagonal is set to one and all other entries of the lower triangle are sampled
-    from a uniform distribution with bounds :code:`[chol_entry_min, chol_entry_max]`.
-    The resulting sparse matrix is then given by :math:`A=LL^\\top`.
+    diagonal is set to one and all other nonzero entries of the lower triangle are sampled from a uniform distribution with bounds :code:`[chol_entry_min, chol_entry_max]`. The resulting sparse matrix is then given by :math:`A=LL^\top`.
```

Comment:
Why is this on a single line? And why doesn't `black` complain about line-length?

Reply:
Not sure why black doesnt format this, possibly because of the Sphinx directives. I fixed the docstring.